gitlab-ci/build_stage.ps1

```diff
@@ -0,0 +1,39 @@
+Param(
+    [Parameter(Mandatory = $True)]
+    [string] $pkg
+)
+
+$ErrorActionPreference="Stop"
+
+cd $PSScriptRoot/../win
+
+${ENV:SCRATCH} = "${ENV:Agent_TempDirectory}/roaster-scratch"
```

Comment:
This dir is specific to gitlab-ci.
I also used it for general purpose script, which is not a good name but doesn't hurt so far.

But for AZP-specific thing, we should definitely create new dir, i.e. `azure-pipelines`.
You can still keep the majority part here, but move things like Agent_TempDirectory there.

Reply:
fixed by adding a script param.